packages/vanilla/src/components/global-nav/side-nav/link/link.js

```diff
@@ -71,7 +73,7 @@ Link._interface =
   Interface.components.GlobalNav.partials.SideNav.partials.SideNavFull.partials.Link;
 Link._defaults = {
   title: 'link',
-  link: '#'
+  link: false
```

Comment:
`link: null`? Otherwise I think your setter will not catch this case.

Reply:
You're right. Will fix.